### DISCUSSION TITLE
[Feature] Add metadata on media upload

### DISCUSSION THREAD

**Discussion by mgogoulos** [mediacms-io/mediacms#605] [Feature Requests]
This will be very useful in cases of multiple file uploads.

Below the upload form, a metadata section could be toggled on/off (off by default) and have title/description/tags/categories specified, plus a button to apply to the new media once hit.

**Comment by mikaelmhansen** [comment]
Agreed. That would be very useful. 
Also please consider (in any location where tags are added) to "auto-match" the chars input by the user, with the list of available tags in the database. That would help the user reuse existing tags rather than risking making new ones.
